pyartcd/pyartcd/pipelines/gen_assembly.py

```diff
@@ -88,8 +92,9 @@ async def run(self):
             if self.custom and (self.auto_previous or self.previous_list or self.in_flight):
                 raise ValueError("Specifying previous list for a custom release is not allowed.")
 
-            candidate_nightlies, latest_nightly = await asyncio.gather(
-                *[self._get_nightlies(), self._get_latest_accepted_nightly()])
+            if not self.skip_get_nightlies:
+                candidate_nightlies, latest_nightly = await asyncio.gather(
```

Comment:
we would need to populate candidate nightlies for the else case?

Reply:
probably just `candidate_nightlies = self.nightlies`

Reply:
thanks for catching that